whois-commons/src/main/resources/applicationContext-commons.xml

```diff
@@ -74,6 +74,12 @@
         <property name="password" value="${whois.db.slave.password}"/>
     </bean>
 
+    <bean id="nrtmDataSource" parent="abstractDataSource">
```

Comment:
Why should this go into commons? It's only needed for nrtmv4?

Reply:
yup, moved to whois-nrtm4 module